match/lib/match/utils.rb

```diff
@@ -18,7 +18,7 @@ def self.fill_environment(params, uuid)
     end
 
     def self.environment_variable_name(params)
-      ["sigh", params[:app_identifier], params[:type]].join("_")
+      ["sigh", params[:app_identifier], params[:type], params[:platform]].join("_")
```

Comment:
Hmm not sure how to handle this one, because tvOS and iOS can have the same app id, but different provisioning profiles, so the platform has to get added so it can be used in the Xcode project.